modules/statics/src/networks.ts

```diff
@@ -535,7 +535,7 @@ class EosTestnet extends Testnet implements AccountNetwork {
 class Hedera extends Mainnet implements AccountNetwork {
   name = 'Hedera';
   family = CoinFamily.HBAR;
-  explorerUrl = 'https://explorer.kabuto.sh/mainnet/transaction/';
+  explorerUrl = 'https://app.dragonglass.me/hedera/search?q=';
```

Comment:
I think, this should be `https://app.dragonglass.me/transactions/`

Reply:
+1

Reply:
I checked a sample transaction, the dragonglass headers are different from transaction id 
attaching a sample to check it 

https://app.dragonglass.me/transactions/008350931710747032564252074

Reply:
[HashScan](https://hashscan.io) works fine with txn hash, check below link for example. It redirects to the correct txn page with ID

[https://hashscan.io/mainnet/transaction/dea1c57c78199f9e576b7de0487f3c3d32dbb65218887662cbc3c3750e90e047ae9d5272402fca40f73a574a90c618ba](https://hashscan.io/mainnet/transaction/dea1c57c78199f9e576b7de0487f3c3d32dbb65218887662cbc3c3750e90e047ae9d5272402fca40f73a574a90c618ba)

Reply:
@ajays97  updated it to hashScan. please review